tests/requirements.py

```diff
@@ -5,6 +5,10 @@
 
 
 class DefaultRequirements(SuiteRequirements):
+    @property
```

Comment:
**mike bayer** (zzzeek) wrote:

the def name should be stated in terms of whatever it is that oracle is not doing, not the DB name itself